lib/handler/status/durations.c

```diff
@@ -35,6 +35,11 @@ struct st_duration_stats_t {
     struct gkc_summary *process_time;
     struct gkc_summary *response_time;
     struct gkc_summary *total_time;
+
+    /**
+     * average event loop latency per worker thread
+     */
+    H2O_VECTOR(uint64_t) evloop_latency_ms;
```

Comment:
Could you please elaborate the reason for not using `gkc_summary` like the values above?

I ask this, because I assume what we want to collect is a histogram of event loop latency, and we use `gkc_summary` for collecting histograms.

Reply:
Sure! we indeed discussed collecting histograms. The reason why this PR exists is to start small by building out the plumbing to publish the data. From there, we can change/grow the stats as we go. That said, using libgkc from the beginning is a legitimate feedback.